src/server.js

```diff
@@ -187,10 +187,21 @@ export async function getBidGroup(req) {
  */
 app.post('/api/bidgroup', server.handleRequest(postBidGroup))
 
+async function delay(milis) {
```

Comment:
this same method exists on `decentraland-commons.utils` as `sleep`, here https://github.com/decentraland/commons/blob/master/src/utils/index.js#L24